taskcluster/glean_taskgraph/__init__.py

```diff
@@ -25,4 +25,11 @@ def _import_modules(modules):
 
 
 def get_decision_parameters(graph_config, parameters):
-    pass
+    if parameters["tasks_for"] == "github-pull-request":
```

Comment:
This reminds me I need to backport this `ci-skip/normal/full` logic in the rest of mobile repos too. We've lost it during the taskgraph migration in all the projects except this one and app-services.